packages/sdk/src/swap/sdk.ts

```diff
@@ -110,6 +114,11 @@ export class SwapSDK {
       ],
     });
     this.dcaEnabled = options.enabledFeatures?.dca ?? false;
+    this.stateChainEnvironmentCache = new AsyncCacheMap({
+      fetch: (_key) => getEnvironment(this.rpcConfig),
+      ttl: 60_000,
```

Comment:
10 mins is probably more than enough
```suggestion
      ttl: 60_000 * 10,
```